README.md

```diff
@@ -1,8 +1,119 @@
-# react-front
+# cnaas-front
+
+## simple standalone dev setup
 
 1. download node
 2. download npm
-3. navigate to the react-front folder
+3. navigate to the cnaas-front folder
 4. run `npm i` to install all the dependencies
 5. run: `npm start` to bundle the js files
 6. navigate to `http://localhost:1234` in browser
+
+
+## full interactive dev setup with docker-compose
+
+This guide will show you how to set up cnaas-nms on your local machine for development.
+The components will run in Docker-containers.
+
+The cnaas-nms frontend will also run inside a Docker container and update interactively with
+every code change.
+
+
+### clone the repositories
+
+Clone the source code for frontend and API. Both should be in the same parent directory.
+
+    git clone https://github.com/SUNET/cnaas-front.git cnaas-front
+    git clone https://github.com/SUNET/cnaas-nms.git cnaas-nms
+
+
+### setup and run docker-compose
+
+Navigate to `cnaas-front/`.
+
+    cd cnaas-front
+
+You will find a `docker-compose.yaml` file here.
+
+This file is used to run docker-compose and build all containers. This takes forever the first time.
+
+    docker-compose up -d
+
+As soon as the built is finished, docker-compose will start the containers.
+
+
+### set up the auth container
+
+A script will set up the auth container for you.
+This script will create a certificate to verify JWTokens issued by the auth server.
+It will also create a user named "cnaas" with password "cnaascnaascnaas".
+
+    docker cp docker/front-dev/setup_auth.sh cnaas-front_cnaas_auth_1:/opt/auth-server-poc/
+    docker exec -t cnaas-front_cnaas_auth_1 /bin/chmod u+x /opt/auth-server-poc/setup_auth.sh
+    docker exec -t cnaas-front_cnaas_auth_1 /opt/auth-server-poc/setup_auth.sh
+
+
+### copy the key for JWT authentication
+
+The public key file is used to verify JWTokens between the auth server and the API.
+`public.pem` has to be copied from the auth container to the API container:
+
+    docker cp cnaas-front_cnaas_auth_1:/opt/auth-server-poc/cert/public.pem .
+    docker cp public.pem cnaas-front_cnaas_api_1:/opt/cnaas/jwtcert/public.pem
+    rm public.pem
+    docker exec -t cnaas-front_cnaas_api_1 /bin/chown root:www-data /opt/cnaas/jwtcert/public.pem
+
+Then, restart the API application.
+
+    docker exec -t cnaas-front_cnaas_api_1 /usr/bin/killall uwsgi
```

Comment:
I don't have killall in my docker but /usr/bin/pkill exists

Reply:
This was my finding too.

Reply:
Talking about unintended side effects:
`killall` is included in the debian package "psmisc" and was likely excluded due to `--no-install-recommends` here: https://github.com/SUNET/cnaas-nms/commit/224a3a2d32f1f9298a59ff31ad6de2b740d97966

I will make a smaller PR to fix this later today.